plugins/stack-overflow/src/search/StackOverflowQuestionsCollator/StackOverflowQuestionsCollator.tsx

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2022 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { IndexableDocument, DocumentCollator } from '@backstage/search-common';
+import { Config } from '@backstage/config';
+import fetch from 'cross-fetch';
+import qs from 'qs';
+import {
+  StackOverflowQuestion,
+  StackOverflowQuestionsRequestParams,
+} from '../../types';
+
+interface StackOverflowDocument extends IndexableDocument {
+  answers: number;
+  tags: string[];
+}
+
+export class StackOverflowQuestionsCollator implements DocumentCollator {
+  protected baseUrl: string;
+  protected requestParams: StackOverflowQuestionsRequestParams;
+  public readonly type: string = 'stack-overflow';
+
+  constructor({
+    config,
+    requestParams,
+  }: {
```

Comment:
remember to unpack variables inside the function body! :) at least for things that end up in the public interface